packages/expo-cli/src/commands/client/selectAdhocProvisioningProfile.js

```diff
@@ -0,0 +1,43 @@
+import ora from 'ora';
+import { runAction, travelingFastlane } from '../build/ios/appleApi/fastlane';
+import { tagForUpdate } from './tagger';
+import log from '../../log';
+
+// XXX: workaround for https://github.com/babel/babel/issues/6262
```

Comment:
How is it possible that we are only experiencing this issue in this file? o.O

Reply:
I hardly believe we need this hack, please check this out:
```
 dsokal@Dominiks-CebulaBook  ~/workspace/expo/expo-cli   master  git grep "export default async function"
packages/expo-cli/src/commands/build/ios/appleApi/authenticate.js:export default async function authenticate(options) {
packages/expo-cli/src/commands/build/ios/appleApi/ensureAppExists.js:export default async function ensureAppExists(appleCtx) {
packages/expo-cli/src/commands/generate-module/configureModule.js:export default async function configureModule(newModulePath, configuration) {
packages/expo-cli/src/commands/generate-module/fetchTemplate.js:export default async function fetchTemplate(destinationPath, template) {
packages/expo-cli/src/commands/generate-module/generateModuleAsync.js:export default async function generateModuleAsync(newModuleProjectDir, options) {
packages/expo-cli/src/commands/generate-module/promptQuestionsAsync.js:export default async function promptQuestionsAsync(suggestedModuleName) {
packages/expo-cli/src/printRunInstructionsAsync.js:export default async function printRunInstructionsAsync() {
packages/xdl/src/detach/installPackagesAsync.js:export default async function installPackagesAsync(
```

Reply:
there are analogous files `selectXXXXXXXXX.js` that have this workaround thing as well, so i've gone ahead and removed them all there as well.

Reply:
Uh, I didn't know we had this chack in other files but thanks for changing all of them.

Reply:
The issue was not that the syntax itself wouldn’t compile, but that referencing the function by name in the same file was broken, causing a runtime error when it was called recursively.

I had this issue about 2 months ago and I don’t think we have updated Babel since then. In any case, this will be fixed once I move these files to TypeScript in the next week or so, so it’s not worth spending too much time on it right now.

Reply:
@fson but there is nothing to spend time on :)
```
 dsokal@Dominiks-CebulaBook  ~/workspace/expo/expo-cli/packages/expo-cli   quin_clientside_creds 
yarn build
yarn run v1.13.0
$ gulp clean && gulp build
[13:30:34] Using gulpfile ~/workspace/expo/expo-cli/packages/expo-cli/gulpfile.js
[13:30:34] Starting 'clean'...
[13:30:34] Finished 'clean' after 43 ms
[13:30:35] Using gulpfile ~/workspace/expo/expo-cli/packages/expo-cli/gulpfile.js
[13:30:35] Starting 'build'...
[13:30:39] Finished 'build' after 4 s
✨  Done in 7.34s.
```

Reply:
@dsokal I think you misunderstood: it’s not a build time error but a runtime errror and it only happens when you try to call the function from the same file (e.g. recursively like this function does).

Reply:
Oh, sorry, it makes sense now :)